src/app/satellite.ts

```diff
@@ -15,9 +15,13 @@ export class Satellite {
    }
 	
 	isSpaceDebris(): boolean {
-		return true;
+		if (this.type === 'Space Debris'){
+			return true;
+		}else{
+			return false;
+		}
```

Comment:
```suggestion
		if (this.type === 'Space Debris'){
			return true;
		}
		return false;
```
Great work with checking for Space Debris! Since your else statement does not involve any other code except a return, you can return false without using an else statement.